cloud/azure/README.md

```diff
@@ -1,7 +1,34 @@
-# Azure Terraform 
+# Azure Terraform Setup
```

Comment:
suggestion (blocking): are there any values generated during these steps that should be considered secrets? if so please specify

Reply:
no there aren't actually it's just specific to your setup

Reply:
ah nice, thanks!